cre.cpp

```diff
@@ -1107,9 +1108,13 @@ static int getWordBoxesFromPositions(lua_State *L) {
 				lastx = wordRect.left;
 			} else {  // word is hyphenated
 				ldomWord word = words[i];
+				int y=-1;
 				for (int j=word.getStart(); j < word.getEnd(); j++) {
-					if (ldomXPointer(word.getNode(), j).getRect(charRect)) {
-						docToWindowRect(tv, charRect);
+					if (ldomXPointer(word.getNode(), j).getRectEX(charRect)) {
+						if(!docToWindowRect(tv,charRect)) continue;
```

Comment:
Could you keep the coding style consistent with the rest of the code in this file? There are couple places that are missing spaces ;)